api/src/pcapi/core/educational/repository.py

```diff
@@ -16,6 +16,7 @@
 from pcapi.core.bookings.utils import convert_booking_dates_utc_to_venue_timezone
 from pcapi.core.educational import exceptions as educational_exceptions
 from pcapi.core.educational import models as educational_models
+from pcapi.core.finance import models as financial_models
```

Comment:
(détail) On a pour habitude de reprendre le nom du package, donc ici `finance` : 

```suggestion
from pcapi.core.finance import models as finance_models
```